### ISSUE TITLE
Help file....

### ISSUE THREAD

**Issue by ToddLa** [yoshisuga/MAME4iOS#105]
The help file needs some serious love.

**Comment by mrjschulte** [comment]
I'll take a stab at this.

**Comment by ToddLa** [comment]
The README too.....
Any reference to the DOSUI should be removed, or re-written to talk about the NativeUI first and DOSUI as a footnote.

**Comment by mrjschulte** [comment]
Do we want to make the help file "context" aware and have specific versions for tvOS vs iOS?

**Comment by mrjschulte** [comment]
@ToddLa What does the dosUI still offer up that we haven't covered in the native ui?

**Comment by ToddLa** [comment]
> Do we want to make the help file "context" aware and have specific versions for tvOS vs iOS?

Easy: Nope
We don’t/can’t show the help file on tvOS

**Comment by ToddLa** [comment]
> @ToddLa What does the dosUI still offer up that we haven't covered in the native ui?

Nothing, just a retro feel. 
So an option is to just take it all out and only cover the GUI UI. 

The Y key on a joystick still brings up a menu where you can add to favorites.ini and delete game files. 
We should turn that off?? Or mention it in the help?

NOTE I am only talking about the root/main MAME UI, it does nothing we don’t do. 

BUT the MAME config menu can do tons of stuff, like key remapping etc

**Comment by mrjschulte** [comment]
> > Do we want to make the help file "context" aware and have specific versions for tvOS vs iOS?
> 
> Easy: Nope
> We don’t/can’t show the help file on tvOS

Ok, then that was that.

**Comment by mrjschulte** [comment]
> > @ToddLa What does the dosUI still offer up that we haven't covered in the native ui?
> 
> Nothing, just a retro feel.
> So an option is to just take it all out and only cover the GUI UI.
> 
> The Y key on a joystick still brings up a menu where you can add to favorites.ini and delete game files.
> We should turn that off?? Or mention it in the help?
> 
> NOTE I am only talking about the root/main MAME UI, it does nothing we don’t do.
> 
> BUT the MAME config menu can do tons of stuff, like key remapping etc

Yes, I'm ok w losing the dosui title but we most definitely want to keep the Mame config menu !
@yoshisuga Thoughts?
